tests/Eccube/Tests/Web/Mypage/RefundRequestControllerTest.php

```
@@ -0,0 +1,316 @@
+            ])
+        );
+
+        $this->assertSame(Response::HTTP_FORBIDDEN, $this->client->getResponse()->getStatusCode());
+    }
+
+    public function testValidationQuantityZero(): void
+    {
+        $this->loginTo($this->Customer);
+        $OrderItem = $this->Order->getProductOrderItems()[0];
+
+        $crawler = $this->client->request(
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$crawler'. (undefined)

(UnusedLocalVariable)